src/main/java/mate/academy/hibernate/relations/service/impl/MovieServiceImpl.java

```diff
@@ -1,16 +1,22 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import jakarta.persistence.EntityNotFoundException;
+import mate.academy.hibernate.relations.dao.MovieDao;
+import mate.academy.hibernate.relations.dao.impl.MovieDaoImpl;
 import mate.academy.hibernate.relations.model.Movie;
 import mate.academy.hibernate.relations.service.MovieService;
+import mate.academy.hibernate.relations.util.HibernateUtil;
 
 public class MovieServiceImpl implements MovieService {
+    private final MovieDao movieDao = new MovieDaoImpl(HibernateUtil.getSessionFactory());
     @Override
     public Movie add(Movie movie) {
-        return null;
+        return movieDao.add(movie);
     }
 
     @Override
     public Movie get(Long id) {
-        return null;
+        return movieDao.get(id).orElseThrow(
+                () -> new EntityNotFoundException("Couldn't get a movie from DB."));
```

Comment:
The error message in the `EntityNotFoundException` is not specific enough. Consider including the `id` of the movie that was not found to make the message more informative. For example: `Couldn't get a movie with id: " + id + " from DB.`